.github/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN install -m 755 /pkgx/pkgm /usr/local/bin/pkgm
 RUN echo 'export PS1="\\[\\033[38;5;63m\\]pkgx\\[\\033[0m\\]\\w $ "' >> /root/.bashrc
 
 FROM debian:buster-slim as stage1
-RUN apt-get update && apt --yes install libc-dev libstdc++-8-dev libgcc-8-dev netbase libudev-dev ca-certificates
+RUN apt-get update && apt --yes install libc-dev libstdc++-8-dev libgcc-8-dev netbase libudev-dev ca-certificates sudo
```

Comment:
how come? Trying to keep it minimal. Did I miss somewhere we need this?

Reply:
That was the last failure with v0.1.0.

Reply:
ah ok, will remove for my own piece of mind

Reply:
Note I want ca-certs gone too (and the rest lol), but that fix was more work for now and wanted some progress.

Reply:
i've been noodling this. embedding certificate info usually isn't a best practice, since it changes with some regularity. but it _should_ be possible. googling suggests:
- create a `Certificate` object in build.rs
- serialize it
- provide it as a build env
- deserialize and consume it using `env!()`
- use `ClientBuilder` to create the reqwest client.